### ISSUE TITLE
Caused by op u'ctc/CTCLoss'

### ISSUE THREAD

**Issue by jjn037** [rizkiarm/LipNet#55]
Caused by op u'ctc/CTCLoss', defined at:
  File "/Users/didi/Documents/LipNet2/training/unseen_speakers/train.py", line 82, in <module>
    train(run_name, 0, 5000, 3, 100, 50, 70, 32, 32)
  File "/Users/didi/Documents/LipNet2/training/unseen_speakers/train.py", line 46, in train
    absolute_max_string_len=absolute_max_string_len, output_size=lip_gen.get_output_size())
  File "/Users/didi/Documents/LipNet2/lipnet/model2.py", line 21, in __init__
    self.build()
  File "/Users/didi/Documents/LipNet2/lipnet/model2.py", line 68, in build
    self.loss_out = CTC('ctc', [self.y_pred, self.labels, self.input_length, self.label_length])
  File "/Users/didi/Documents/LipNet2/lipnet/core/layers.py", line 7, in CTC
    return Lambda(ctc_lambda_func, output_shape=(1,), name=name)(args)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/keras/engine/topology.py", line 554, in __call__
    output = self.call(inputs, **kwargs)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/keras/layers/core.py", line 659, in call
    return self.function(inputs, **arguments)
  File "/Users/didi/Documents/LipNet2/lipnet/core/loss.py", line 12, in ctc_lambda_func
    return K.ctc_batch_cost(labels, y_pred, input_length, label_length)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/keras/backend/tensorflow_backend.py", line 3264, in ctc_batch_cost
    sequence_length=input_length), 1)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/tensorflow/python/ops/ctc_ops.py", line 145, in ctc_loss
    ctc_merge_repeated=ctc_merge_repeated)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/tensorflow/python/ops/gen_ctc_ops.py", line 164, in _ctc_loss
    name=name)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 763, in apply_op
    op_def=op_def)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2327, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/Users/didi/anaconda3/envs/py2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1226, in __init__
    self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): sequence_length(0) <= 70
	 [[Node: ctc/CTCLoss = CTCLoss[ctc_merge_repeated=true, preprocess_collapse_repeated=false, _device="/job:localhost/replica:0/task:0/cpu:0"](ctc/Log, ctc/ToInt64, ctc/ToInt32_2, ctc/ToInt32_1)]]

How does the error happening
